packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1473,6 +1478,12 @@ class Navigator extends StatefulWidget {
   /// Defaults to false.
   final bool reportsRouteUpdateToEngine;
 
+  /// Whether or not the navigator and it's new topmost route should request focus
+  /// when the new route is pushed onto the navigator.
+  ///
+  /// Defaults to true.
+  final bool takeFocus;
```

Comment:
I would call this `requestFocus` or `shouldRequestFocus`, since that is in line with the naming of `requestFocus()` on the focus node.

Reply:
Done.